actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -1,9 +1,12 @@
 require 'thread_safe'
+require 'action_view/path_set'
 
 module ActionView
   class DependencyTracker # :nodoc:
     @trackers = ThreadSafe::Cache.new
 
+    mattr_accessor(:view_paths) { ActionView::PathSet.new }
```

Comment:
I think the view context has a controller reference, but I think the tracker run before the context is created. But worth to investigate.